src/plone/restapi/indexers.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from plone.restapi.behaviors import IBlocks
+from plone.indexer.decorator import indexer
+from plone.app.contenttypes.indexers import SearchableText
+import six
+
+
+def _extract_text(block):
+    result = ""
+    for paragraph in block.get("text").get("blocks"):
+        text = paragraph["text"]
+        if six.PY2:
+            if isinstance(text, six.text_type):
+                text = text.encode("utf-8", "replace")
+            if text:
+                result = " ".join((result, text))
+        else:
+            result = " ".join((result, text))
+    return result
+
+
+@indexer(IBlocks)
+def SearchableText_blocks(obj):
+    std_text = SearchableText(obj)
```

Comment:
I am not sure that calling `SearchableText` is good here.
I think this is going to clash with the `collective.dexteritytextindexer` behavior.

Reply:
@ale-rt would you mind elaborating how the indexer could clash with c.dexteritytextindexer?

Reply:
@ale-rt @tisto this really caused problems. See: #1764